mgo.go

```diff
@@ -65,12 +66,15 @@ func (c *Client) Store(cr CrawlingResult) error {
 		return fmt.Errorf("Client is not connected")
 	}
 	summary := summary(cr.Employees)
-	agmi := AgencyMonthlyInfo{AgencyID: cr.AgencyID, Month: cr.Month, Year: cr.Year, Crawler: cr.Crawler, Employee: cr.Employees, Summary: summary}
-	_, err := c.col.ReplaceOne(context.TODO(), bson.D{{Key: "aid", Value: cr.AgencyID}, {Key: "year", Value: cr.Year}, {Key: "month", Value: cr.Month}}, agmi, options.Replace().SetUpsert(true))
+	backup, err := c.sc.backup(cr.Files)
 	if err != nil {
-		return fmt.Errorf("error trying to update mongodb with value {%+v}: %q", agmi, err)
+		return fmt.Errorf("error trying to get Backup files: %v", err)
```

Comment:
"error trying to back up files"

Poderia também imprimir os arquivos que estão sendo feito backup, por favor?